src/Insightly/Item/Organisation.php

```diff
@@ -174,26 +174,28 @@ public static function jsonUnSerialize($json)
      */
     public function toInsightly()
     {
-        $addresses = [];
-        foreach ($this->addresses as $address) {
-            $addresses[] = $address->toInsightly();
-        }
-
-        $contactInfo = [];
-        foreach ($this->contactInfo as $info) {
-            $contactInfo[] = $info->toInsightly();
-        }
-
         $data = parent::toInsightly();
 
         $data += [
             'ORGANISATION_ID' => $this->getId(),
             'ORGANISATION_NAME' => $this->getName(),
             'BACKGROUND' => $this->getBackground(),
-            'ADDRESSES' => $addresses,
-            'CONTACTINFOS' => $contactInfo,
         ];
 
+        foreach ($this->contactInfo as $contactInfo) {
+            if ($contactInfo->getType() === ContactInfo::TYPE_EMAIL) {
+                $data['EMAIL_ADDRESS'] = $contactInfo->getDetail();
+                break;
+            }
+        }
+
+        if (count($this->addresses) > 0)
+        {
+            /** @var Address $address */
+            $address = $this->addresses[0];
+            $data =\array_merge($data, $address->toInsightly());
```

Comment:
Small nit: Missing a space here after the `=` (I feel like a linter now).
You can also import the function, or just leave out the `\`. I don't think we usually add the `\` for global functions in other repo's like UDB3 etc.